client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -46,15 +46,27 @@ define(['utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view',
         /** Build form */
         _buildForm: function(options) {
             var self = this;
+            var req_message = self._templateRequirements(options)[0].textContent;
```

Comment:
The `_templateRequirements` should not need to build a dom, since only the text seems to be used.